docs/blog/es-tooling.md

```diff
@@ -0,0 +1,90 @@
+---
+title: ES Tooling - community lead e18e-focused tooling
+author:
+  - name: James Garbutt and Pascal Schilp
+sidebar: false
+date: 2024-07-22
+head:
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: og:title
+      content: es tooling - community lead e18e-focused tooling
+  - - meta
+    - property: og:url
+      content: https://e18e.dev/blog/es-tooling
+  - - meta
+    - property: og:description
+      content: ES Tooling - a community lead, e18e-focused collection of tools for the ecosystem
+---
+
+# ES Tooling
+
+_July 22, 2024_
+
+The e18e initiative has seen huge growth since launch. We've seen hundreds of performance contributions upstream to popular projects, and the creation of many community lead tools.
+
+A notable example of this is the [ES Tooling](https://github.com/orgs/es-tooling/repositories) community GitHub organisation.
+
+The ES Tooling organisation has been created to hold various e18e-focused tools and forks maintained by the community.
+
+If you haven't yet encountered e18e, read more in the [guide](https://e18e.dev/guide/#why-e18e) and [join the community](https://chat.e18e.dev/).
+
+## A home for e18e-focused tools
+
+Over the last few months, many of us have collaborated and connected through the e18e community. The ES tooling organisation is a result of that - a place for us to store and own forks, tools, and other code bases we're working on together.
+
+A few popular projects that were running solo until now, have come together to create this:
+
+- [ecosystem cleanup](https://github.com/43081j/ecosystem-cleanup).
+- [module replacements](https://github.com/es-tooling/module-replacements).
+- [module replacements codemods](https://github.com/thepassle/module-replacements-codemods)
+- [fd-package-json](https://github.com/es-tooling/fd-package-json)
+
+The ES Tooling org predates the creation of the e18e initiative. In case you're wondering about why these efforts won't live under the e18e org, we think that it is better to keep this organization focused on [fostering the space to connect us](https://chat.e18e.dev/), and maintaining a curation of ecosystem performance projects.
```

Comment:
I'm a bit confused of the `fostering the space to connect us` part. Is this referring to e18e or es tooling? I assume the latter as before it mentions "this organization focused on" which I assume refers to es tooling.

If so, I thought e18e is the place for `fostering the space to connect us` and not directly related to "this organization"?

Reply:
this was a previous PR comment i just committed. better to ask @patak-dev 

he was trying to point out why this stuff doesn't live in e18e's github org

Reply:
Is this more clear?
```suggestion
The ES Tooling org predates the creation of the e18e initiative. In case you're wondering about why these efforts won't live under the e18e org, we think that it is better to keep the e18e organization focused on [fostering a space to connect us](https://chat.e18e.dev/), and as a way to curate ecosystem performance projects through these docs, blog, and social accounts.
```